test/lib/java-wrapper.test.ts

```diff
@@ -1,9 +1,9 @@
 import * as path from 'path';
 import {
   getClassPerJarMapping,
-  getTargets,
   getCallGraphGenCommandArgs,
 } from '../../lib/java-wrapper';
+import { getTargets } from '../../lib/index';
```

Comment:
Small thing is that this file doesn't test `/index` - may be worth moving?

Reply:
Addressed.